client/src/app/site/pages/meetings/pages/participants/pages/mandate-check/components/mandate-check-list/mandate-check-list.component.ts

```diff
@@ -0,0 +1,223 @@
+    }
+
+    public getTotalCount(genderId?: Id): number {
+        if (genderId) {
+            const genderEntry = this.genderEntryMap.get(genderId);
+            return (genderEntry?.present ?? 0) + (genderEntry?.absent ?? 0);
+        }
+        return this.presentUserIds.length + this.absentUserIds.length;
+    }
+
+    public getPercent(genderId?: Id): number {
+        const total = this.getTotalCount(genderId);
+        const part = genderId ? this.genderEntryMap.get(genderId)?.present : this.presentUserIds.length;
+        return part / total;
+    }
+}
+
+@Component({
+    selector: 'os-mandate-check-list',
+    imports: [
+        TranslateModule,
+        HeadBarModule,
+        ReactiveFormsModule,
+        MatButtonModule,
+        MatCardModule,
+        MatFormFieldModule,
+        MatIconModule,
+        MatProgressBarModule,
+        MatSelectModule
+    ],
+    templateUrl: './mandate-check-list.component.html',
+    styleUrl: './mandate-check-list.component.scss',
+    changeDetection: ChangeDetectionStrategy.Default
+})
+export class MandateCheckListComponent extends BaseMeetingComponent implements OnDestroy, OnInit {
+    public structureLevels = [];
+    public entriesObservable = new BehaviorSubject<MandateCheckEntry[]>([]);
+    public entries: MandateCheckEntry[] = [];
+    public groups: ViewGroup[] = [];
+    public participants: ViewUser[] = [];
+    public genderIds: Id[] = [2, 1, 3, 4];
+    public genders: any[] = [];
+    public selectedGroups: Id[] = [];
+    public form: UntypedFormGroup = null;
+    public toggleMap = new Map<Id, boolean>();
+
+    public constructor(
+        private participantRepo: ParticipantControllerService,
+        private structureLevelRepo: StructureLevelRepositoryService,
+        private groupRepo: GroupControllerService,
+        private genderRepo: GenderControllerService,
+        private formBuilder: UntypedFormBuilder,
+        private cd: ChangeDetectorRef
+    ) {
+        super();
+    }
+
+    public ngOnInit(): void {
+        this.subscriptions.push(
+            this.structureLevelRepo.getViewModelListObservable().subscribe(strLvls => {
+                this.structureLevels = strLvls;
+                this.updateToggleMap();
+                this.cd.markForCheck();
+            })
+        );
+        this.subscriptions.push(
+            this.participantRepo.getViewModelListObservable().subscribe(participants => {
+                this.participants = participants;
+                this.updateEntries();
+                this.cd.markForCheck();
+            })
+        );
+        this.subscriptions.push(
+            this.groupRepo.getViewModelListObservable().subscribe(groups => (this.groups = groups))
+        );
+
+        this.form = this.formBuilder.group({ groups: [] });
+        this.form.valueChanges.subscribe(values => {
+            this.selectedGroups = values.groups;
+            this.updateEntries();
+            this.cd.markForCheck();
+        });
```

Comment:
subscription.push accepts multiple Subscriptions, you can just handle all three of them in the same call.